packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -658,9 +659,9 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
       query: string | TemplateStringsArray | sqlTemplateTag.Sql,
       ...values: sqlTemplateTag.RawValue[]
     ) {
-      const request = (transactionId?: number, runInTransaction?: boolean) => {
+      const request = (txId?: number, inTx?: boolean, otelCtx?: Context) => {
         try {
-          const promise = this.$executeRawInternal(runInTransaction ?? false, transactionId, query, ...values)
+          const promise = this.$executeRawInternal(txId, inTx, otelCtx, query, ...values)
```

Comment:
What motives difference in param naming here vs. $executeRawInternal?

Reply:
Good point, I should rename the other instances too. I feel like `txId` and `inTx` are explicit enough with their shorter names.